src/main/java/frc/robot/subsystems/Bling.java

```diff
@@ -0,0 +1,35 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class Bling extends SubsystemBase {
+
+
+  private static Bling INSTANCE = new Bling();
```

Comment:
Suggest you follow the Singleton pattern and move the instantiation ("new Bling()") inside the getInstance or constructor

Reply:
I originally did that, but when I got Nathan to have a look at it, he said that I could move it there.